RELEASE_NOTES

```diff
@@ -3,9 +3,42 @@ For more detailed information, please see the git log.
 
 These release notes can also be consulted at http://easybuild.readthedocs.org/en/latest/Release_notes.html.
 
+v2.8.2 (July 12th 2016)
+-----------------------
+
+bugfix release
+- various small enhancements, including:
+    - add support for rst output for --list-* and --avail-* (#1339)
```

Comment:
@caylo: OK, this is solid progress in favor of an automated manpage, thanks:
https://github.com/hpcugent/easybuild-framework/issues/271#issuecomment-9657096